src/cloud/basic-information/starter-develop-deploy-workflow.md

```diff
@@ -203,14 +203,14 @@ To learn more, see [Deploy your store]({{ site.baseurl }}/cloud/live/stage-prod-
 
 ### Push to Master / Production {#pro}
 
-When you push to the `master` branch, you are pushing to the Production environment. Complete configuration and testing activities in the Production environment like you did in the Staging environment with one important difference. In the Production environment use live credentials for during configuration and testing. The moment you go live and launch, customers must be able to complete purchases and administrators should be able to manage your live store.
+When you push to the `master` branch, you are pushing to the Production environment. Complete configuration and testing activities in the Production environment like you did in the Staging environment with one important difference. In the Production environment use live credentials for during configuration and testing. The moment you launch your site,  customers must be able to complete purchases and administrators should be able to manage your live store.
 To learn more, see [Deploy your store]({{ site.baseurl }}/cloud/live/stage-prod-live.html).
 
-### Go live {#go-live}
+### Site launch {#go-live}
```

Comment:
I would love to get rid of the anchor, too. I think `site-launch` is a better one anyway.